test/service/test_manifest.py

```diff
@@ -788,3 +791,41 @@ def test_manifest_format_validation(self):
         url = self.base_url + '/manifest/files?format=invalid-type'
         response = requests.get(url)
         self.assertEqual(400, response.status_code, response.content)
+
+    @mock_sts
+    @mock_s3
+    def test_manifest_content_disposition_header(self):
+        self._index_canned_bundle(("f79257a7-dfc6-46d6-ae00-ba4b25313c10", "2018-09-14T133314.453337Z"))
+        for filters, expected_name, name_file in (
+                ({'project': {'is': ['Single of human pancreas']}}, 'Single of human pancreas ', True),
+                ({'project': {'is': ['Single of human pancreas', 'Mouse Melanoma']}}, 'hca-manifest-', False),
+                ({}, 'hca-manifest-', False)
+        ):
+            storage_service = StorageService()
+            storage_service.create_bucket()
+            for single_part in True, False:
+                with self.subTest(filters=filters, single_part=single_part):
+                    with mock.patch.object(type(config), 'disable_multipart_manifests', single_part):
+                        params = {
+                            'filters': json.dumps(filters),
+                            'format': 'full'
+                        }
+                        filters = AbstractService.parse_filters(params.get('filters'))
+                        estd = ElasticTransformDump()
+                        url = estd.transform_manifest(params.get('format', 'full'), filters).headers['Location']
+                        query = urlparse(url).query
+                        content_dispositions = parse_qs(query).get('response-content-disposition')
+                        if content_dispositions:
```

Comment:
There is the potential that a bug in production code is never true. The condition should not be driven by actual state, but by expected state. Feel free to talk to me if that's not clear.